packages/wrangler/src/__tests__/utils/log-file.test.ts

```diff
@@ -0,0 +1,126 @@
+import { existsSync, readdirSync, readFileSync } from "node:fs";
+import { join } from "node:path";
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
+import { appendToDebugLogFile, debugLogFilepath } from "../../utils/log-file";
+import { runInTempDir } from "../helpers/run-in-tmp";
+
+vi.mock("../../logger", () => ({
+	logger: {
+		error: vi.fn(),
+		debug: vi.fn(),
+		console: vi.fn(),
+	},
+}));
```

Comment:
Remember, you shouldn't be mocking the logger. Instead, use our existing utilities for that purpose